doc/content/verification_and_validation/ver-1fc.md

```diff
@@ -0,0 +1,72 @@
+# ver-1fc
+
+# Conduction in composite structure with constant surface temperatures
+
+The third heat transfer problem (after [ver-1fa](ver-1fa.md) and [ver-1fb](ver-1fb.md)) studies heat transfer through a composite with constant surface temperatures.
+The composite is a 40 cm thick layer of Copper (Cu) followed by a 40 cm layer of iron (Fe) [!cite](ambrosek2008verification).
+The temperature of both layers is initially 0 K, but at time t = 0, the outside face of the Cu is held at 600 K while the outside face of the Fe is maintained at 0 K.
+The thermal conductivity of Cu and Fe are set to 401 W/m/K and 80.2 W/m/K, respectively. The TMAP7 documentation does not specify the materials density $\rho$ or the specific heat $C_p$, but the TMAP7 input file list $\rho C_p = 3.4392 \times 10^6$ J$\cdot$m$^{-3}\cdot$K$^{-1}$ for Cu and $3.5179 \times 10^6$ J$\cdot$m$^{-3}\cdot$K$^{-1}$ for Fe [!cite](ambrosek2008verification). To match these values, TMAP8 uses $\rho = 8940.0$ kg/m$^{3}$ and $C_p =  384.70$ J$\cdot$kg$^{-1}\cdot$K$^{-1}$ for Cu and $\rho = 7860.0$ kg/m$^{3}$ and $C_p = 447.57$ J$\cdot$kg$^{-1}\cdot$K$^{-1}$ for Fe.
```

Comment:
I'm not sure about 

> The thermal conductivity of Cu and Fe is set to 401 W/m/K and 80.2 W/m/K.

Since it's the thermal conductivity for both Cu and Fe. 
In any case, good catches for (at least) the other two suggestions.

Reply:
If we write 'thermal conductivities' then 'are' is appropriate, if we write 'thermal conductivity' then 'is' should be used. The auxiliary verb applies to the thermal conductivity (the noun that's getting an action of being 'set').

Reply:
I'll do: `The thermal conductivities of Cu and Fe are set to 401 W/m/K and 80.2 W/m/K`